python/dgl/nn/pytorch/explain/gnnexplainer.py

```diff
@@ -35,28 +35,41 @@ class GNNExplainer(nn.Module):
         The learning rate to use, default to 0.01.
     num_epochs : int, optional
         The number of epochs to train.
+    edge_size : float, optional
```

Comment:
Is this some sort of budgets? It is quite counter-intuitive to have a higher "size" value that results in fewer edges.

Reply:
It's the weight of a regularizer term for penalizing edge size. Any naming suggestions?

Reply:
Updated